client/deposits/strings.ts

```diff
@@ -5,16 +5,21 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+
+import type { DepositStatus } from 'wcpay/types/deposits';
+
 export const displayType = {
 	deposit: __( 'Deposit', 'woocommerce-payments' ),
 	withdrawal: __( 'Withdrawal', 'woocommerce-payments' ),
 };
 
-export const displayStatus = {
+export const displayStatus: Record< DepositStatus, string > = {
```

Comment:
I've explicitly linked this to `DepositStatus` to help communicate the intended purpose of this mapping and help prevent potential bugs that may occur if we leave out a deposit status value.